sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -271,6 +275,11 @@ public ContactFacadeEjb(ContactService service, UserService userService) {
 		super(Contact.class, ContactDto.class, service, userService);
 	}
 
+	@Override
+	public AbstractCoreAdoService<Contact> getEntityService() {
```

Comment:
please remove this method, we already have a protected service field on AbstractBaseEjb

Reply:
removed